changelog/db.changelog-1.1.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="John Watson" id="1">
+    <tagDatabase tag="v1.1.0" />
+  </changeSet>
+  <changeSet author="John Watson" id="2">
+    <createTable tableName="organisations">
+      <column name="sbi" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="organisationsId_pkey" />
+      </column>
+      <column name="addressLine1" type="VARCHAR(127)" />
+      <column name="addressLine2" type="VARCHAR(127)" />
+      <column name="addressLine3" type="VARCHAR(127)" />
+      <column name="city" type="VARCHAR(127)" />
+      <column name="county" type="VARCHAR(127)" />
+      <column name="postCode" type="VARCHAR(8)" />
+      <column name="emailAddress" type="VARCHAR(127)" />
+      <column name="frn" type="BIGINT" />
+      <column name="name" type="VARCHAR(127)" />
+      <column name="updated" type="TIMESTAMP" />
+    </createTable>
+  </changeSet>
+  <changeSet author="John Watson" id="3">
+    <createTable tableName="calculations">
+      <column name="calculationId" type="INTEGER">
+        <constraints nullable="false" primaryKey="true" primaryKeyName="calculationId_pkey" />
+      </column>
+      <column name="sbi" type="INTEGER" />
+      <column name="frn" type="BIGINT" />
+      <column name="calculationDate" type="TIMESTAMP" />
+      <column name="invoiceNumber" type="VARCHAR(30)" />
+      <column name="scheme" type="VARCHAR(20)" />
+      <column name="updated" type="TIMESTAMP" />
+    </createTable>
+  </changeSet>
+  <changeSet author="John Watson" id="4">
+    <createTable tableName="fundings">
+      <column name="fundingId" type="INTEGER" autoIncrement="true" >
+        <constraints nullable="false" primaryKey="true" primaryKeyName="fundingsId_pkey" />
+      </column>
+      <column name="fundingCode" type="VARCHAR(10)" />
+      <column name="calculationId" type="INTEGER" />
+      <column name="areaClaimed" type="DECIMAL(19,4)" />
+      <column name="rate" type="DECIMAL(18,6)" />
```

Comment:
tht's even more wild tht Cross Border's area size and precision 😆

Reply:
Indeed!  So this one is taken from the BPS Claim Statement Generator.  Some of the rates of reductions are calculated at six decimal places so have replicated the same so we can cover those schemes too.

18 might be too big though in hindsight as we know the rate isn't that high.  Will drop down